tests/test_datacard.py

```diff
@@ -116,17 +116,21 @@ def test_datacard_file_duration_distribution(
         f"{db.name}-file-durations.png",
     )
     assert not os.path.exists(image_file)
-    expected_distribution_str = f"{expected_min:.1f} s .. {expected_max:.1f} s"
+    if expected_min == expected_max:
+        expected_distribution_str = f"{expected_max:.1f} s"
```

Comment:
Below you assert that the image file does exist when there is a distribution and all values are not equal. 
Is there a reason to not assert the opposite here (You would expect here that no image file exists)?

Reply:
In line 118 we do:

```python
assert not os.path.exists(image_file)
```
As `Datacard.sphinx_src_dir` is not set, the image file should always be absent at this stage, independent if durations are all equal or not.

Reply:
I wonder whether `expected_distribution_str` will be fully intuitive in the case that there is no variatioin in the distribution distribution:

`0, duration distribution: 0.0 s .. 0.0 s`

Or would one drop the keyword ''range' somehow? Or one could  elaborate on the interpretation in the docstring.

Reply:
What are you referring to with `"range"`?

The expected outcome for your case from above with 0 files and no duration variation would be:

```
0, duration distribution: 0.0 s
```

Reply:
> In line 118 we do:
> 
> ```python
> assert not os.path.exists(image_file)
> ```
> 
> As `Datacard.sphinx_src_dir` is not set, the image file should always be absent at this stage, independent if durations are all equal or not.

Sorry, overlooked this. This can be seen as resolved.

Reply:
I believe that `60, duration distribution: each file is 600.0 s` would be more self-explanatory.
While the string is slightly longer, I can see a benefit.

Reply:
I changed it accordingly.

Reply:
Great. Resolving this.